component/constraint/model/CableModel.h

```diff
@@ -31,9 +31,9 @@
 #ifndef SOFA_COMPONENT_CONSTRAINTSET_CABLEMODEL_H
 #define SOFA_COMPONENT_CONSTRAINTSET_CABLEMODEL_H
 
-#include <sofa/core/behavior/Constraint.h>
-#include <sofa/core/behavior/MechanicalState.h>
-#include <sofa/core/behavior/OdeSolver.h>
+//#include <sofa/core/behavior/Constraint.h>
```

Comment:
Commenting dead feature is a bad practice. Don't commit it. 
A small explanation why:
https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f